### ISSUE TITLE
Add Entity Clone

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#129]
We have requests from current customers to allow the duplication of a page. I think [Entity Clone](https://www.drupal.org/project/issues/entity_clone) does this and supports entity reference revisions (paragraphs and nesting).

We could use this to create default unpublished templates for users!

**Comment by joewhitsitt** [comment]
Particularly this issue, the ability to set permissions on a entity type per entity type basis: https://www.drupal.org/project/entity_clone/issues/2931143

**Comment by joewhitsitt** [comment]
Good thing we were not early adopters. There are serveral issues with paragraphs currently.
> Bumping to critical, in our project we are extensively using (nested) Paragraphs, and this bug causes errors left and right.

**Comment by richardbporter** [comment]
Will we still need this after #1779? I don't think the use case will be as strong since a default layout can be set for all pages.

**Comment by pyrello** [comment]
We are still looking for a good solution to allow for different types of page templates, so if this might fill that role, I'd be interested in checking it out. I talked about this a little bit in this comment: https://github.com/uiowa/uiowa/issues/1550#issuecomment-647015619

**Comment by joewhitsitt** [comment]
Another issue that might affect our solution (slider/collection paragraphs within a block within a page) -  https://www.drupal.org/project/entity_clone/issues/3050027

Might also be worth giving a look: https://www.drupal.org/project/quick_node_clone (mentioned in the above issue)

**Comment by briand44** [comment]
closing this in favor of #4223
